tests/install_vm.py

```diff
@@ -83,6 +85,25 @@ def parse_args():
     return parser.parse_args()
 
 
+def detect_url_os(url):
+    """Given a repository URL, guess an OS type and version."""
+    cmd = ["osinfo-detect", "-f", "env", "-t", "tree", url]
```

Comment:
Please also add `libosinfo` package into the "Libvirt backend" section of `tests/README.md`.

Reply:
@matusmarhefka `libosinfo` is already used and required by `virt-install`, so if you have `virt-install` you already have `libosinfo`. In addition, the python exception takes care of cases where it's not present and the repository is simply not set up (still allowing some level of testing or manual repo setup).

Are you sure you want me to put it in `README.md`? .. Maybe as "optional"?

Reply:
You are right. In that case there is no need to put the package there.